src/m365/spfx/commands/project/project-github-workflow-add.ts

```diff
@@ -110,6 +110,10 @@ class SpfxProjectGithubWorkflowAddCommand extends BaseProjectCommand {
       throw new CommandError(`Couldn't find project root folder`, SpfxProjectGithubWorkflowAddCommand.ERROR_NO_PROJECT_ROOT_FOLDER);
     }
 
+    if (!args.options.overwrite) {
+      await this.warn(logger, `It is recommended to use --overwrite option and it will become the default behavior in next major release.`);
```

Comment:
Let's explain why it's recommended. Also, let's change to active voice tone and say that it's us who recommend it. Otherwise folks might wonder where the recommendation is coming from and why.

Reply:
would something like this work:
```
We recommend always using the --overwrite option and it will become the default behavior in next major release. Overwriting your sppkg file on every deployment is required to make continuous delivery of the latest version of your app which is the aim of the continuous delivery pipeline.
```

Reply:
Perfect!